DEVELOP.md

```diff
@@ -90,37 +91,44 @@ Run `gulp peg` to update the PEG.js generated grammar after modifying CQL.
 Run `gulp prod` to update all of the above (BioDalliance, PEG.js grammar, etc.)
 
 
-### Python
+### 6. Start CycleDash
 
-If `USE_RELOADER` is True in your ENV.sh, then you'll get automatic
-code-reloading with the Flask server and JS/CSS reloading via livereload and
-gulp's livereload plugin.
-
-To test the workers locally, you'll need to install and run rabbitmq during
-development. For example, on Mac OS X, you can do this via:
+To start the application server:
 
 ```bash
-brew install rabbitmq
+gulp prod
+./run.sh
+```
+
+
+### 7. Run RabbitMQ (and keep it running)
+
+To test the workers locally, you'll need to install and run rabbitmq during development. For example, on Mac OS X, you can do this via:
```

Comment:
Similarly, try to stick to the 80 char line limit.

(Or else we could switch the entire file to use no line length limit, instead?)